packages/x-tree-view/src/internals/plugins/useTreeViewSelection/useTreeViewSelection.utils.ts

```diff
@@ -1,41 +1,41 @@
 import { TreeViewInstance } from '../../models';
-import { UseTreeViewNodesSignature } from '../useTreeViewNodes';
+import { UseTreeViewItemsSignature } from '../useTreeViewItems';
 
 /**
  * This is used to determine the start and end of a selection range so
- * we can get the nodes between the two border nodes.
+ * we can get the items between the two border items.
  *
- * It finds the nodes' common ancestor using
+ * It finds the items' common ancestor using
  * a naive implementation of a lowest common ancestor algorithm
  * (https://en.wikipedia.org/wiki/Lowest_common_ancestor).
- * Then compares the ancestor's 2 children that are ancestors of nodeA and NodeB
- * so we can compare their indexes to work out which node comes first in a depth first search.
+ * Then compares the ancestor's 2 children that are ancestors of itemA and ItemB
+ * so we can compare their indexes to work out which item comes first in a depth first search.
  * (https://en.wikipedia.org/wiki/Depth-first_search)
  *
- * Another way to put it is which node is shallower in a trémaux tree
+ * Another way to put it is which item is shallower in a trémaux tree
  * https://en.wikipedia.org/wiki/Tr%C3%A9maux_tree
  */
 export const findOrderInTremauxTree = (
-  instance: TreeViewInstance<[UseTreeViewNodesSignature]>,
-  nodeAId: string,
-  nodeBId: string,
+  instance: TreeViewInstance<[UseTreeViewItemsSignature]>,
+  itemAId: string,
+  itemBId: string,
 ) => {
-  if (nodeAId === nodeBId) {
-    return [nodeAId, nodeBId];
+  if (itemAId === itemBId) {
+    return [itemAId, itemBId];
   }
 
-  const nodeA = instance.getNode(nodeAId);
-  const nodeB = instance.getNode(nodeBId);
+  const itemA = instance.getNode(itemAId);
```

Comment:
Values returned by `instance.getNode` should probably be called `node`
I think we couldrename this method `instance.getItemMetaData`, rename the related interfaces and the returned value.